core/src/main/java/com/backblaze/b2/client/credentialsSources/B2CredentialsFromEnvironmentSource.java

```diff
@@ -8,7 +8,7 @@
 
 /**
  * This B2CredentialsSource reads the credentials from two environment variables:
- *   B2_ACCOUNT_ID
+ *   B2_APPLICATION_KEY_ID
```

Comment:
i think this change means that people who run programs which use this class will have their programs stop working until they change their environment variables.  is that necessary?

Reply:
If they upgrade to the newest SDK once this is merged I believe that they need to change their environment variables names, which is documented.

 I'm not sure how to answer your question about being necessary. The goal behind the app key renaming is to bring clarity to the name. My intention with this change was to bring b2-java-sdk users up to speed on the same terminology that we want everyone to use.

Reply:
briang & i talked and he's taking a stab at maintaining compatibility.